Competition2017/src/telemetry/InfluxDBConnection.java

```diff
@@ -0,0 +1,48 @@
+package telemetry;
+
+import java.util.concurrent.TimeUnit;
+
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Point;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.StringProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class InfluxDBConnection {
+
+    protected final BooleanProperty isEnabledProp;
+    protected final StringProperty influxAddressProp;
+    
+    protected final InfluxDB influxDB;
+    protected final String dbName = "XbotRobotData";
+    
+    @Inject
+    public InfluxDBConnection(XPropertyManager propMan) {
+        isEnabledProp = propMan.createPersistentProperty("InfluxDB logging enabled", false);
+        influxAddressProp = propMan.createPersistentProperty("Influx address", "http://10.4.88.100:8086");
```

Comment:
Good question, there are several reasons in favor from my perspective:

- Doesn't use up any of the (limited) system resources on the rio (memory, cpu, disk)
- Doesn't require any additional setup/installing of packages in the roborio